lib/validators/modelValidator.ts

```diff
@@ -790,10 +792,9 @@ export class ModelValidator extends SpecValidator<SpecValidationResult> {
           // validation of path segment.
           parameterValue = parameterValue.replace(/\//gi, "")
 
-          // replacing scheme that may cause validator failed when x-ms-parameterized-host enbaled
-          if (hostTemplate) {
-            // TODO: we can get the scheme from parameterValue if the useSchemePrefix is setting false in the x-ms-parameterized-host,
-            // then check if it can match to the swagger scheme.
+          // replacing scheme that may cause validator failed when x-ms-parameterized-host enbaled & useSchemePrefix enabled
+          // because if useSchemePrefix enabled ,the parameter value in x-ms-parameterized-host should not has the scheme (http://|https://)
+          if (useSchemePrefix) {
```

Comment:
useSchemePrefix is true by default if it's not set. 
https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-parameterized-host

Can you also add test case when useSchemePrefix = false?

Reply:
sure, that's important

Reply:
already has the test case in  describe : "for parameters in x-ms-parameterized-host"